psvpfsparser/FilesDbParser.cpp

```diff
@@ -348,6 +342,14 @@ bool FilesDbParser::parseFilesDb(std::ifstream& inputStream, std::vector<sce_ng_
          return false;
       }
 
+      //bad block with error or unknown format
+      bool is_bad_block = false;
+      if (block.header.nFiles > 9)
```

Comment:
So the error was caused by nFiles being greater than 9 ?
Can you document title ids that have this issue? Just add the comment to the code.

Reply:
added

https://github.com/motoharu-gosuto/psvpfstools/blob/94471aefbc4e991bc2981f1489237df9c7ff44f2/psvpfsparser/FilesDbParser.cpp#L351-L359

Reply:
Perfect. Both usage of constant and provided information.